compare.py

```diff
@@ -5,8 +5,9 @@
 import matplotlib as mpl
```

Comment:
Some of these imports can be removed after the file split, e.g., the imports for plotting libraries are only needed in `charts.py`. VSCode/black should give good hints for this; look for squiggles by the unused imports at the top of each python file, and just remove the unused ones.